feature-libs/checkout/root/checkout-root.module.ts

```diff
@@ -70,8 +65,8 @@ export function defaultCheckoutComponentsConfig() {
   ],
   providers: [
     ...interceptors,
-    provideDefaultConfig(defaultCheckoutRoutingConfig),
-    provideDefaultConfig(defaultCheckoutConfig),
+    // provideDefaultConfig(defaultCheckoutRoutingConfig),
```

Comment:
had to comment this out after type augmentation as it failed to compile afterwards

Reply:
Added back. The build doesn't fail for me?
Also, in repl PR, we have a flag to enable the old checkout in env.ts.
I enabled it, and ran `yarn build`, and the build doesn't fail.

Reply:
weird

Reply:
Brian (me) will take a look at it